book/plugins/general-abilities/js2native/readme.md

```diff
@@ -190,30 +212,34 @@ Please make sure to map any PII data you send to the naming convention of the fi
 * Phone
 * SocialIDs
 
- ### Native player
+ ## Native player
```

Comment:
@snooky23 you have unnecessary space before `## Native Player` and the line below.

Reply:
fixed